### ISSUE TITLE
Show feature value for explain_prediction

### ISSUE THREAD

**Issue by lopuhin** [TeamHG-Memex/eli5#126]
For numeric features it can be valuable to know the feature value: for example, some feature can be either positive or negative depending on it's value, and it can be made more obvious if we add a third column "Value" to the feature weights table.
On the other hand, it's possible (and rather easy) to implement it outside of eli5, so I'm not sure if it's worth including.

**Comment by kmike** [comment]
Yeah, makes sense. It is not only about sign, but also about feature scale. Do we show features multiplied by weights now, or just the weights?

**Comment by lopuhin** [comment]
We show features multiplied by weights, and in case of trees we also show final contribution, so features with different scales will have the same weight if they have equal contribution.

I was also thinking that " (missing)" that we are discussing in #123 is also a special case of feature value. But I don't have a good idea how it fits together.
